tools/autograd/derivatives.yaml

```diff
@@ -297,6 +297,9 @@
 - name: cholesky(Tensor self, bool upper=False) -> Tensor
   self: cholesky_backward(grad, upper, result)
 
+- name: cholesky_mod(Tensor self, bool upper=False, *, Tensor? jitter=None) -> (Tensor result, Tensor jitter_out)
+  self: cholesky_mod_backward(grad, upper, result, jitter_out)
+
```

Comment:
Maybe it makes sense to tag the gradient wrt to jitter as `non_differentiable`?

Reply:
right, makes sense.